Sources/Sentry/SentryTimeToDisplayTracker.m

```diff
@@ -142,7 +142,7 @@ - (void)framesTrackerHasNewFrame:(NSDate *)newFrameDate
         if (!_waitForFullDisplay) {
             [SentryDependencyContainer.sharedInstance.framesTracker removeListener:self];
 #    if SENTRY_TARGET_PROFILING_SUPPORTED
-            if (!SentrySDK.options.enableContinuousProfiling) {
+            if (SentrySDK.options.profilesSampleRate != nil) {
```

Comment:
`m`: Why can't we use `isContinuousProfilingEnabled` here?

Reply:
The team has decided to completely remove that option. It wasn't really my preference, but it allows it to work with less configuration code.

Reply:
ah, i missed this one